rampwf/prediction_types/tests/test_detection_predictions.py

```diff
@@ -49,10 +49,10 @@ def test_combine_ignore_none():
 
     # first item: 2 out of 3 match -> confidence of 2/3
     # second item: ignore None, so 2 out of 2 match -> confidence of 2/2 = 1
-    expected = np.empty(2, dtype=object)
-    expected[:] = [[(2. / 3, 1, 1, 1)], [(1, 3, 3, 1)]]
-    assert_allclose(expected[0], y_pred_combined[0])
```

Comment:
Here the issue is that otherwise, y_pred_combined[0] is,
```
[array([0.6666666666666666, 1.0, 1.0, 1.0], dtype=object)]
```
and such objects can no longer be compared against an array in numpy >1.24.0. Previously with <1.24.0 it was,
```
[array([0.66666667, 1.        , 1.        , 1.        ])]
```